docs/documentation/upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,16 @@
 == Migration Changes
 
+<<<<<<< HEAD
```

Comment:
There are some issues in this file, which cause the documentation is impossible to build.

Could you please fix it and remove "conflict markers" from this file?

Also I suggest to manually build documentation on your laptop and checking it looks as expected before pushing it into the PR.

Reply:
Yes, I made the changes and then tried to build the documentation locally on my laptop and it's working fine.